syntax = "proto3";

option go_package = "github.com/cilium/proxy/go/cilium/api;cilium";

package cilium;

message L7Policy {
  // Path to the unix domain socket for the cilium access log.
  string access_log_path = 1;

  // HTTP response body message for 403 status code.
  // If empty, "Access denied" will be used.
  string denied_403_body = 3;
}
